### ISSUE TITLE
ModuleNotFoundError: No module named 'fs.archive.sevenzipfs.SevenZipFS'

### ISSUE THREAD

**Issue by mattalxndr** [althonos/fs.archive#8]
Something seems to be broken in the loading of the 7z extra. I tested another extra (iso) and it seems to be ok.

Here is my test:

```shell
~ % python -m venv venv
~ % source venv/bin/activate
(venv) ~ % pip install "fs.archive[all]"
Collecting fs.archive[all]
Using cached fs.archive-0.7.0-py2.py3-none-any.whl (31 kB)
Collecting fs~=2.2
Using cached fs-2.4.14-py2.py3-none-any.whl (133 kB)
Collecting six~=1.10
Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Requirement already satisfied: setuptools>=38.3.0 in ./venv/lib/python3.10/site-packages (from fs.archive[all]) (58.1.0)
Collecting pycdlib~=1.8
Using cached pycdlib-1.12.0-py2.py3-none-any.whl (208 kB)
Collecting iocursor~=0.1
Using cached iocursor-0.1.2-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (47 kB)
Collecting py7zr~=0.17
Using cached py7zr-0.17.2-py3-none-any.whl (68 kB)
Collecting appdirs~=1.4.3
Using cached appdirs-1.4.4-py2.py3-none-any.whl (9.6 kB)
Collecting pytz
Using cached pytz-2021.3-py2.py3-none-any.whl (503 kB)
Collecting multivolumefile>=0.2.3
Using cached multivolumefile-0.2.3-py3-none-any.whl (17 kB)
Collecting pyzstd>=0.14.4
Using cached pyzstd-0.15.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.8 MB)
Collecting pybcj>=0.5.0
Using cached pybcj-0.5.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (47 kB)
Collecting pyppmd>=0.17.0
Using cached pyppmd-0.17.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (127 kB)
Collecting brotli>=1.0.9
Using cached Brotli-1.0.9-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (2.7 MB)
Collecting texttable
Using cached texttable-1.6.4-py2.py3-none-any.whl (10 kB)
Collecting pycryptodomex>=3.6.6
Using cached pycryptodomex-3.12.0-cp35-abi3-manylinux2010_x86_64.whl (2.0 MB)
Installing collected packages: six, pytz, appdirs, texttable, pyzstd, pyppmd, pycryptodomex, pybcj, multivolumefile, fs, brotli, pycdlib, py7zr, iocursor, fs.archive
Successfully installed appdirs-1.4.4 brotli-1.0.9 fs-2.4.14 fs.archive-0.7.0 iocursor-0.1.2 multivolumefile-0.2.3 py7zr-0.17.2 pybcj-0.5.0 pycdlib-1.12.0 pycryptodomex-3.12.0 pyppmd-0.17.3 pytz-2021.3 pyzstd-0.15.0 six-1.16.0 texttable-1.6.4
WARNING: You are using pip version 21.2.4; however, version 21.3.1 is available.
You should consider upgrading via the '/home/matt/venv/bin/python -m pip install --upgrade pip' command.
(venv) ~ % cat > test.py <<EOF
heredoc> from fs import open_fs
heredoc> from fs.archive import open_archive
heredoc> my_fs = open_fs(u'temp://')
heredoc> with open_archive(my_fs, u'test.zip') as archive:
heredoc>        print(type(archive))
heredoc> with open_archive(my_fs, u'test.iso') as archive:
heredoc>        print(type(archive))
heredoc> with open_archive(my_fs, u'test.7z') as archive:
heredoc>        print(type(archive))
heredoc> EOF
(venv) ~ % python test.py
<class 'fs.archive.zipfs.ZipFS'>
<class 'fs.archive.isofs.ISOFS'>
Traceback (most recent call last):
File "/home/matt/test.py", line 8, in <module>
with open_archive(my_fs, u'test.7z') as archive:
File "/home/matt/venv/lib/python3.10/site-packages/fs/archive/opener.py", line 56, in open_archive
archive_opener = entry_point.load()
File "/home/matt/venv/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2450, in load
return self.resolve()
File "/home/matt/venv/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2456, in resolve
module = __import__(self.module_name, fromlist=['__name__'], level=0)
ModuleNotFoundError: No module named 'fs.archive.sevenzipfs.SevenZipFS'
```

And my environment:

```shell
(venv) ~ % python --version
Python 3.10.1
(venv) ~ % pip --version
pip 21.2.4 from /home/matt/venv/lib/python3.10/site-packages/pip (python 3.10)
(venv) ~ % pip list | grep fs\.
fs              2.4.14
fs.archive      0.7.0
WARNING: You are using pip version 21.2.4; however, version 21.3.1 is available.
You should consider upgrading via the '/home/matt/venv/bin/python -m pip install --upgrade pip' command.
```

**Comment by althonos** [comment]
@mattalexx : That's actually because of a typo in `setup.cfg` :zipper_mouth_face: I'll publish a patch right now.

**Comment by althonos** [comment]
Should be fixed in `v0.7.1`.

**Comment by mattalxndr** [comment]
Thanks
